tests/FakeItEasy.Tests/ArgumentConstraintManagerExtensions/StringStartsWithWithComparisonTypeTests.cs

```diff
@@ -0,0 +1,52 @@
+namespace FakeItEasy.Tests.ArgumentConstraintManagerExtensions
+{
+    using System;
+    using System.Collections.Generic;
+    using FakeItEasy.Tests.TestHelpers;
+    using Xunit;
+
+    public class StringStartsWithWithComparisonTypeTests
+        : ArgumentConstraintTestBase<string>
+    {
+        protected override string ExpectedDescription => "string that starts with \"ABC\"";
+
+        public static IEnumerable<object?[]> InvalidValues()
+        {
+            return TestCases.FromObject(
+                "foo",
+                "bar",
+                "biz",
+                "baz",
+                "lorem ipsum",
+                (object?)null);

```

Comment:
Nitpick: could include values that contain "abc" but do not start with it. Feel free to apply or ignore the suggestion!

Reply:
It's a fair point.